app/src/main/java/mozilla/lockbox/store/SettingStore.kt

```diff
@@ -46,7 +46,7 @@ open class SettingStore(
 
     private lateinit var preferences: SharedPreferences
     @RequiresApi(Build.VERSION_CODES.O)
-    private lateinit var autofillManager: AutofillManager
+    open lateinit var autofillManager: AutofillManager
```

Comment:
👾

This is making `AutofillManager` part of the public interface for this store. If I understand correctly, this is to make it easier to test. 

I think probably the best way of solving this problem is to mock out the autofill manager and context, then inject it using `injectContext`. The [`SettingStoreTest` shows this][1] quite nicely.

[1]: https://github.com/mozilla-lockbox/lockbox-android/blob/5c321cff1031f75f0e6be5b0b731ca3b846e776e/app/src/test/java/mozilla/lockbox/store/SettingStoreTest.kt#L195

Reply:
Ahh I think I tried every way of mocking the AutofillManager and SettingStore except `whenCalled(context.getSystemService(AutofillManager::class.java)).thenReturn(autofillManager)`. Thanks!